src/pages/index.js

```diff
@@ -75,22 +131,64 @@ addCardButton.addEventListener("click", () => {
   cardPopup.open();
 });
 
-// CLASSES
-const profileEditPopup = new PopupWithForm(profileEditModal, handleFormSubmit);
+profileImageEditButton.addEventListener("click", () => {
+  profileImageEditPopup.open();
+})
+
+
+
+// ~~~~~~~~~~~~~~~~~ API ~~~~~~~~~~~~~~~~~ //
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "ba6f95f2-7e3a-4192-b9e4-9397fc618de9",
+    "Content-Type": "application/json"
+  }
+});
+
+let cardSection;
+
+api.getInitialCards()
+.then((cards) => {cardSection = new Section({
+  items: cards,
+  renderer: createCard,
+}, cardListEl);
+cardSection.renderItems();
+})
+.catch((err) => {console.error(err)})
+
+api.getUserInfo()
+.then((userInfo) => {
+  api.editUserInfo({title: userInfo.name, description: userInfo.about});
```

Comment:
Needs correction: there is no need to update the profile info right after receiving it. You should do that only after submitting the form.

Reply:
Hello, regarding this correction,
I'm calling `api.getUserInfo()` and `api.editUserInfo()` when the page opens so that the response from the server will render onto the site. 
I have `handleProfileEditSubmit()` that handles the userInfo information after a submitted form on line 62. 

Is there another way I should go about this or is this correction resolved? Thank you

![image_2023-12-17_130557457](https://github.com/Lochbird/se_project_aroundtheus/assets/130000036/58f2c4e4-2e4d-42cd-98e2-a407f83195a1)

Reply:
The response from the server is rendered when you call `setUserInfo` on the line 171. There is no need to update the info on the server here, where you receive it, since it's already actual. Please remove line 170